js/client-chat.js

```diff
@@ -1736,12 +1751,17 @@
 		},
 		comparator: function (a, b) {
 			if (a === b) return 0;
+
+			var aUser = this.room.users[a];
+			var bUser = this.room.users[b];
+			if (aUser.away !== bUser.away) return aUser.away - bUser.away;
+
 			var aRank = (
-				Config.groups[(this.room.users[a] ? this.room.users[a].charAt(0) : Config.defaultGroup || ' ')] ||
+				Config.groups[(aUser ? aUser.name.charAt(0) : Config.defaultGroup || ' ')] ||
```

Comment:
```suggestion
				Config.groups[aUser ? aUser.name.charAt(0) : Config.defaultGroup || ' '] ||
```

(parens are useless here)